store/sqlstore/upgrade.go

```diff
@@ -365,8 +365,10 @@ func UpgradeDatabaseToVersion471(sqlStore SqlStore) {
 }
 
 func UpgradeDatabaseToVersion48(sqlStore SqlStore) {
+
 	//TODO: Uncomment the following condition when version 4.8.0 is released
 	//if shouldPerformUpgrade(sqlStore, VERSION_4_7_0, VERSION_4_8_0) {
+	//	sqlStore.CreateColumnIfNotExists("Teams", "LastTeamIconUpdate", "bigint", "bigint", "0")
```

Comment:
Oh, this shouldn't be commented. Just the surrounding if statement. We want it to always try creating that column on master until we're ready to actually lock the version 4.8.0 schema
```go
	//TODO: Uncomment the following condition when version 4.8.0 is released
	//if shouldPerformUpgrade(sqlStore, VERSION_4_7_0, VERSION_4_8_0) {
	sqlStore.CreateColumnIfNotExists("Teams", "LastTeamIconUpdate", "bigint", "bigint", "0")
	//	saveSchemaVersion(sqlStore, VERSION_4_8_0)
	//}
```

Reply:
done :)